package.json

```diff
@@ -61,5 +61,10 @@
     "singleQuote": true,
     "arrowParens": "avoid",
     "trailingComma": "all"
+  },
+  "pnpm": {
+    "onlyBuiltDependencies": [
```

Comment:
Needed for modern SvelteKit apps for some reason I don't fully understand; needs to be in the root package.json to take effect